content/stub.xhtml

```diff
@@ -837,7 +837,7 @@
             <span title="{{fullDate}}">{{date}}</span>
           </span>
           <span class="replyLinkWrapper">
-            <a href="javascript:" class="link replyMainActionLink" style="display:none"></a>
+            | <a href="javascript:" class="link replyMainActionLink"></a>
```

Comment:
I don't know much about xhtml, but does that '| ' at the beginning belong there?